### DISCUSSION TITLE
Transfer table for POS

### DISCUSSION THREAD

**Discussion by amrhelw** [phplime/Qrexorder#7] [Ideas]
Hello,
I suggest a transfer table for POS, so if someone changes his table or he wants to pay for another table beside him.

**Comment by eskays76** [comment]
It will be a good option for Order Management to Transfer Table Orders from one to another Table Number, It happens in real practice sometimes Customers place an order from One Table and then move to another Table for some reason. 

And also have an option to Merge Different Table Orders for combined billing, sometimes a large family member visits a restaurant to have a family dinner or lunch and they can't sit on a single table so they sit on 2-3 different tables, but Order place from a Host person and he will pay the complete bill for all.

**Comment by phplime** [comment]
This will be added in the next update

**Reply by amrhelw** [reply]
thanks alot.
